services/storage/src/simcore_service_storage/modules/celery/client.py

```diff
@@ -55,10 +62,20 @@ async def send_task(
         ):
             task_uuid = uuid4()
             task_id = build_task_id(task_context, task_uuid)
-            self._celery_app.send_task(task_name, task_id=task_id, kwargs=task_params)
-            await self._task_store.set_task(
-                task_id, TaskData(status=TaskState.PENDING.name)
+            task_metadata = task_metadata or TaskMetadata()
+            self._celery_app.send_task(
+                task_name,
+                task_id=task_id,
+                kwargs=task_params,
+                queue=task_metadata.queue,
```

Comment:
task_metadata can be None and defaults to None, how can this even work? Is your linter not complaining about this?

Reply:
ok forget that I could finally open it in vscode and saw that it gets created from scratch if None.